### ISSUE TITLE
Lint report hex error in selector

### ISSUE THREAD

**Issue by joaomilho** [brigade/scss-lint#36]
Lint is reporting a hex error in a selector. E.g.:

``` css
#defaultPaymentTable td:first-child {
  font-size: 16px !important;
}
```

``` shell
$ scss-lint file.scss
... - Hexadecimal color codes should be lowercase and in 3-digit form where possible
```
